test/e2e/common_test.go

```diff
@@ -476,6 +476,17 @@ func (p *PodmanTestIntegration) RunNginxWithHealthCheck(name string) (*PodmanSes
 	return session, session.OutputToString()
 }
 
+// RunContainerWithNetworkTest runs the fedoraMinimal curl with the specified network mode.
+func (p *PodmanTestIntegration) RunContainerWithNetworkTest(mode string) *PodmanSessionIntegration {
+	var podmanArgs = []string{"run"}
+	if mode != "" {
+		podmanArgs = append(podmanArgs, "--network", mode)
+	}
+	podmanArgs = append(podmanArgs, fedoraMinimal, "curl", "-k", "-o", "/dev/null", "www.podman.io")
```

Comment:
I'd like this to be `http://www.podman.io:80` to make it 100% unambiguous. See comments below.

Reply:
Done.